tbapy/main.py

```diff
@@ -315,15 +315,29 @@ def team_profiles(self, team):
         return [Profile(raw) for raw in self._get('team/%s/social_media' % self.team_key(team))]
 
     @_check_modified
-    def team_status(self, team, event):
+    def team_status(self, team=None, event=None, year=None):
         """
         Get status of a team at an event.
 
         :param team: Team whose status to get.
         :param event: Event team is at.
+        :param year: Year of events to return.
         :return: Status object.
         """
-        return Status(self._get('team/%s/event/%s/status' % (self.team_key(team), event)))
+        if event:
+            if team:
+                return Status(self._get('team/%s/event/%s/status' % (self.team_key(team), event)))
+            else:
```

Comment:
I would prefer that we reduce the branch complexity of this code; since we immediately return within each condition you can remove the `else`s and de-indent the blocks within, since it will only be continued to if the initial condition isn't met. Do this for all the `else`s.